aws-ecr-replicationconfiguration/aws-ecr-replicationconfiguration.json

```diff
@@ -0,0 +1,125 @@
+            "required": [
+                "Destinations"
+            ]
+        },
+        "Destinations": {
+            "type": "array",
+            "minItems": 1,
+            "maxItems": 25,
+            "items": {
+                "$ref": "#/definitions/ReplicationDestination"
+            },
+            "description": "An array of objects representing the details of a replication destination.",
+            "required": [
+                "ReplicationDestination"
+            ]
+        },
+        "ReplicationDestination": {
+            "type": "object",
+            "properties": {
+                "Region": {
+                    "$ref": "#/definitions/Region"
+                },
+                "RegistryId": {
+                    "$ref": "#/definitions/RegistryId"
+                }
+            },
+            "description": "An array of objects representing the details of a replication destination.",
+            "required": [
+                "Region",
+                "RegistryId"
+            ]
+        },
+        "RegistryId": {
+            "type": "string",
+            "description": "The account ID of the destination registry to replicate to.",
+            "pattern": "^[0-9]{12}$"
+        },
+        "Region": {
+            "description": "A Region to replicate to.",
+            "type": "string",
+            "pattern": "[0-z9a-z-]{2,25}"
+        }
+    },
+    "properties": {
+        "ReplicationConfiguration": {
+            "type": "object",
+            "properties": {
+                "Rules": {
+                    "$ref": "#/definitions/Rules"
+                }
+            },
+            "description": "An object representing the replication configuration for a registry.",
+            "required": [
+                "Rules"
+            ]
+        },
+        "RegistryId": {
+            "type": "string",
+            "description": "The RegistryId associated with the aws account."
+        }
+    },
+    "required": [
+        "ReplicationConfiguration"
+    ],
+    "primaryIdentifier": [
+        "/properties/RegistryId"
+    ],
+    "readOnlyProperties": [
+        "/properties/RegistryId"
+    ],
+    "additionalProperties": false,
+    "handlers": {
+        "create": {
+            "permissions": [
+                "ecr:DescribeRegistry",
+                "ecr:PutReplicationConfiguration",
+                "iam:CreateServiceLinkedRole"
+            ]
+        },
+        "read": {
+            "permissions": [
+                "ecr:DescribeRegistry"
+            ]
+        },
+        "update": {
+            "permissions": [
+                "ecr:DescribeRegistry",
+                "ecr:PutReplicationConfiguration",
+                "iam:CreateServiceLinkedRole"
+            ]
+        },
+        "delete": {
+            "permissions": [
+                "ecr:DescribeRegistry",
+                "ecr:PutReplicationConfiguration",
+                "iam:CreateServiceLinkedRole"
```

Comment:
Do we require this permission for `delete`?

Reply:
As discussed, this is required.